examples/html_tables/table_sample.py

```diff
@@ -0,0 +1,36 @@
+import os, sys, json, codecs
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(os.path.join(os.path.dirname(__file__), '../..'))
+from etk.etk import ETK
+from etk.extractors.table_extractor import TableExtractor, EntityTableDataExtraction
+
+sample_html = json.load(codecs.open('./table_data.json', 'r')) # read sample file from disk
+
+etk = ETK()
+doc = etk.create_document(sample_html, mime_type="text/html", url="http://ex.com/123")
+
+my_table_extractor = TableExtractor()
+
+d = doc.select_segments("$.raw_content")[0]
+root = doc.select_segments("$")[0]
+
+tables = doc.invoke_extractor(my_table_extractor, d)
+for t in tables:
+    root.store_extractions([t], t.tag)
+
+table_data_extractor = EntityTableDataExtraction()
+table_data_extractor.add_glossary(etk.load_glossary("./resources/address_dict.txt"), "address")
```

Comment:
This code implies that I can add new glossaries after extracting one table, and then when I extract a second table, it will use the new glossaries. Is this correct?

Reply:
The tables are extracted from the document at line 17. After all the tables are extracted, the `table_data_extractor` is created and desired glossaries are added. Then at line 32 the `table_data_extractor` is run on all the tables in the document (which have been extracted before).